packages/framework/src/hooks/useThemeContext.tsx

```diff
@@ -13,26 +13,21 @@ export const CustomThemeContext = createContext<CustomTheme>({});
 
 export const useThemeScope = (): {
   theme: EnsembleThemeModel | undefined;
-  themeName: string;
+  themeName?: string;
```

Comment:
Why is this optional? Does flutter return null or default if there is a single theme?